com.unity.formats.fbx/Editor/ExportModelEditorWindow.cs

```diff
@@ -101,6 +101,77 @@ protected float FbxExtLabelWidth
 
         protected abstract ExportOptionsSettingsSerializeBase SettingsObject { get; }
 
+        // Helper functions for persisting the Export Settings for the session
+        protected virtual string SessionStoragePrefix { get { return "FbxExporterOptions_{0}"; } }
+        protected virtual void StoreSettingsInSession()
+        {
+            var settings = SettingsObject;
+            var json = EditorJsonUtility.ToJson(settings);
+            SessionState.SetString(string.Format(SessionStoragePrefix, nameof(settings)), json);
+
+            // store Fbx Save Paths
+            if (m_fbxSavePaths == null)
+            {
+                return;
+            }
+            var joinedPaths = string.Join(",", m_fbxSavePaths);
+            SessionState.SetString(string.Format(SessionStoragePrefix, nameof(m_fbxSavePaths)), joinedPaths);
+            SessionState.SetInt(string.Format(SessionStoragePrefix, nameof(SelectedFbxPath)), SelectedFbxPath);
+        }
+
+        protected virtual void RestoreSettingsFromSession(ExportOptionsSettingsSerializeBase defaults)
+        {
+            var settings = SettingsObject;
+            var json = SessionState.GetString(string.Format(SessionStoragePrefix, nameof(settings)), EditorJsonUtility.ToJson(defaults));
+            if (!string.IsNullOrEmpty(json))
+            {
+                EditorJsonUtility.FromJsonOverwrite(json, settings);
+            }
+        }
+
+        public virtual void ClearSessionSettings()
+        {
+            var settings = SettingsObject;
+            SessionState.EraseString(string.Format(SessionStoragePrefix, nameof(settings)));
+            SessionState.EraseString(string.Format(SessionStoragePrefix, nameof(m_fbxSavePaths)));
+            m_fbxSavePaths = null;
+            SessionState.EraseInt(string.Format(SessionStoragePrefix, nameof(SelectedFbxPath)));
+            SelectedFbxPath = 0;
+        }
+
+        [SerializeField]
+        private List<string> m_fbxSavePaths;
+        internal List<string> FbxSavePaths
+        {
+            get
+            {
+                if (m_fbxSavePaths == null)
+                {
+                    // Try to restore from session, fall back to Fbx Export Settings
+                    var pathStr = SessionState.GetString(string.Format(SessionStoragePrefix, nameof(m_fbxSavePaths)), ExportSettings.instance.GetJoinedFbxSavePaths());
```

Comment:
I think you've got this code twice, can you refactor it (and do the same fix of actually storing all the paths separately so you don't have trouble with funny-named paths)